### ISSUE TITLE
USA Saves

### ISSUE THREAD

**Issue by Kambushed12312** [CryZe/WindWakerSaves#4]
is there a savefile made in the USA version of WW , I want it in the earth temple with medli

**Comment by LyykaPaws** [comment]
Are you looking for a 100% save, or an Any% save?

**Comment by LyykaPaws** [comment]
If you're looking for a 100% save, try save #13 in the 100% folder. If you're looking for an Any% save, try #11 in the Any% (Chasetopher) folder.

Let me know if you need anything else.
/Ryan

**Comment by Nitropurasu** [comment]
Hello, do you have USA save for the Forsaken Fortress 2? The one that starts after you get the Master Sword. I lost my save and I dont wanna to start from scratch. Please help me :(

**Comment by LyykaPaws** [comment]
I need to know if you want a 100% save, or any% first, please.

**Comment by Nitropurasu** [comment]
Any. But if you can give me 100% save, that would be cool

**Comment by LyykaPaws** [comment]
Sorry for the late reply. I've been busy with school, so I haven't had a chance since last week to come check on the page. Unfortunately, for 100% saves, the closest I could get you is to Hyrule (First Time) without skipping stuff. So, hopefully, you don't mind having to work a little bit to get up to the Forsaken Fortress (2nd). It shouldn't take too long (hopefully... :P)
Here's the saves:

[Any% #8 - Islet & Forsaken Fortress (Second Time) (Click to Download)](https://github.com/CryZe/WindWakerSaves/raw/master/Any%25%20%28Chasetopher%29/08_Islet%2BEnterFF2.gci)
[100% #9 - Hyrule First Time (Click to Download)](https://github.com/CryZe/WindWakerSaves/raw/master/100%25/9-Hyrule1.gci)

**Comment by Nitropurasu** [comment]
Sorry but I completed the game already. Thank you anyway

**Comment by kuupe** [comment]
Got a USA save file for before you meet Jabun? Found couple of good saves here, but they seem to be PAL as they don't work for me.

**Comment by kuupe** [comment]
Any kind of file is good really, I'm not picky. As long as it's the USA version.

**Comment by SkyThaBro** [comment]
will these save files work on dolphin (gamecube emulator)

**Comment by LyykaPaws** [comment]
Yes, these save files will work in Dolphin. They are saves FILES, not save states, so you're gonna want to import them to your GC Card in Dolphin, through the GC Card Manager.

**Comment by SkyThaBro** [comment]
i imported a save from reddit from a random dude and it worked but when i imported one of your saves i got this, and the wind waker doesn't see that.
![import](https://cloud.githubusercontent.com/assets/26172710/23710721/004f8228-041e-11e7-8144-954c2b5f7614.png)

**Comment by Zeldafan1** [comment]
I need USA save file for the earth temple and wind temple

**Comment by advillabotomite** [comment]
I also need a US or EU save for after the Wind Temple

**Comment by PW5190** [comment]
I'm on the hunt for a save that has two Quest Logs both 100% completed. Quest Log 1 on the Second Quest, and Quest Log 2 on the Normal Quest. While Quest Log 1 can be saved after completing everything and defeating Ganon, Quest Log 2 should have everything completed, with Ganon not yet defeated. This is for the NTSC-U version, and both Quest Logs should use Link's default name (just name them Link).
